views/beers.hbs

```diff
@@ -0,0 +1,5 @@
+ <div class="beers-container">
+        {{#each beerList}}
```

Comment:
here instead of using the `beerList` you should use `beers` as it is the name of the propriety that you are sending to this view.